ratis-shell/src/main/java/org/apache/ratis/shell/cli/sh/command/ElectionCommand.java

```diff
@@ -29,7 +30,7 @@
 public class ElectionCommand extends AbstractParentCommand {
   private static final List<Function<Context, AbstractRatisCommand>> SUB_COMMAND_CONSTRUCTORS
       = Collections.unmodifiableList(Arrays.asList(
-      TransferCommand::new, PauseCommand::new, ResumeCommand::new));
+      TransferCommand::new, PauseCommand::new, ResumeCommand::new, StepDownCommand::new));
```

Comment:
Let's move StepDownCommand next to TransferCommand.
```
      TransferCommand::new, StepDownCommand::new, PauseCommand::new, ResumeCommand::new));
```
Also, we should use LinkedHashMap in AbstractParentCommand so that it can preserve the order.
```
+++ b/ratis-shell/src/main/java/org/apache/ratis/shell/cli/sh/command/AbstractParentCommand.java
@@ -32,7 +33,9 @@ public abstract class AbstractParentCommand extends AbstractRatisCommand{
     super(context);
     this.subs = Collections.unmodifiableMap(subCommandConstructors.stream()
         .map(constructor -> constructor.apply(context))
-        .collect(Collectors.toMap(Command::getCommandName, Function.identity())));
+        .collect(Collectors.toMap(Command::getCommandName, Function.identity(),
+            (a, b) -> {throw new IllegalStateException("Found duplicated commands: " + a + " and " + b);},
+            LinkedHashMap::new)));
   }
```
Without the change above, the sub commands will be printed out randomly.
```
Usage: ratis sh [generic options]
	 [election [resume] [stepDown] [transfer] [pause]]         
	 [group [list] [info]]                                     
	 [peer [add] [setPriority] [remove]]                       
	 [snapshot [create]]                                       
```

Reply:
done